basilisk/ipc/windows_ipc.py

```
@@ -0,0 +1,145 @@
+		Args:
+			pipe_name: Name of the named pipe (without '\\\\.\\pipe\\' prefix)
+		"""
+		super().__init__(pipe_name)
+		self.pipe_name = f"\\\\.\\pipe\\{pipe_name}"
+
+	def _run_server(self):
+		"""Run the named pipe server loop with concurrent connection handling."""
+		while self.running:
+			try:
+				# Create the named pipe with multiple instances support
+				pipe_handle = win32pipe.CreateNamedPipe(
```

Flake8 (7.2.0)

[warning] indentation contains tabs

(W191)